### DISCUSSION TITLE
SVG support media library

### DISCUSSION THREAD

**Discussion by michaeljcoyne** [area17/twill#1495] [General]
Hi I am using the media library  with glide config.

When trying to upload an SVG image I get 

`Intervention\Image\Exception\NotReadableException:` Unsupported image type image/svg. GD driver is only able to decode JPG, PNG, GIF, BMP or WebP files. in file /Users/michaelcoyne/Documents/Sites/twill/vendor/intervention/image/src/Intervention/Image/Gd/Decoder.php on line `

I am a bit confused as it says in the docs that SVG is supported by I also see GD driver doesnt support it. I take it Glide does not support SVG? Or is there a config to get this to work?

https://spectrum.chat/twill/help/cant-upload-svgs~99e0cce9-0028-46e8-b5e7-797515896755
https://www.google.com/search?client=firefox-b-d&q=intervention+image+svg

**Comment by haringsrob** [comment]
Hey @michaeljcoyne,

You should be able to upload svg's just fine. However, you will not be able to use any form of image manipulation on them.

What did you do to receive the error? Maybe we can better handle it.

Cheers
